### DISCUSSION TITLE
No emails

### DISCUSSION THREAD

**Discussion by myanesdiazl9** [YetiForceCompany/YetiForceCRM#14327] [How to] [answered]
Community, could you help me find out why an email history is not being stored?
I appreciate the help
![image](https://user-images.githubusercontent.com/68311999/110073964-bd35a100-7d56-11eb-9b7d-1834c030bd09.png)

**Comment by rskrzypczak** [comment, marked as answer]
https://www.youtube.com/watch?v=0JBckYE9ZuI&list=PLX0lktP3Kukg_Da-TaEBys-aON68Yy2vh&index=11

**Reply by myanesdiazl9** [reply]
Trank you 
This is my current configuration 
![image](https://user-images.githubusercontent.com/68311999/110137064-ce0e0300-7da6-11eb-9113-bd8a0facf5c2.png)
What am I doing wrong?

**Reply by rskrzypczak** [reply]
Watch the video once again and set what is shown, e.g. actions etc.

**Reply by myanesdiazl9** [reply]
Thank you very much, the configuration worked for me.
